irony-cdb.el

```diff
@@ -63,6 +63,20 @@ for files that it cannot handle."
   :type '(repeat function)
   :group 'irony-cdb)
 
+(defcustom irony-cdb-search-directory-list '("." "build")
```

Comment:
oh,I thought it was from cmake upstream,  but LLVM matters as well. Also, at least irony
supports its own recommendations now... :)